src/components/leaderboard/LeaderboardTable.tsx

```diff
@@ -81,6 +81,9 @@ export const LeaderboardTable = ({ data }: Props) => {
           totalPages={2}
           onClickNextPage={handleNextPage}
           onClickPrevPage={handlePrevPage}
+          onPageClick={function (pageNumber: number): void {
+            throw new Error("Function not implemented.");
```

Comment:
why throw error, should we set tooltip like coming soon or disable?

Reply:
fixed